readme.md

```diff
@@ -106,7 +108,8 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+// CREATE READ/SHOW UPDATE DESTROY- don't recall.
+
 
 ```
```

Comment:
Let me know if there's any way I can help out with Express review. But what the question is asking here is to set up some Express routes. All they need to do is send a string as a response - not looking for any database commands here. So the answer would look something like this...

```js
var express = require("express");
var app = express();

app.get("/", function(request, response){
  response.send("SHOW");
});
app.post("/", function(request, response){
  response.send("CREATE");
});
app.put("/:id", function(request, response){
  response.send("UPDATE");
});
app.delete("/:id", function(request, response){
  response.send("DELETE");
});
```